client/layout/guided-tours/tours/jetpack-plugin-updates-tour.js

```diff
@@ -0,0 +1,91 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import { query } from 'layout/guided-tours/positioning';
+
+export const JetpackPluginUpdatesTour = makeTour(
+	<Tour name="jetpackPluginUpdates" version="20180611">
+		{ /* Phantom step, wait for placeholder to disappear, then advance */ }
+		<Step
+			name="init"
+			target=".plugin-item.is-placeholder"
+			onTargetDisappear={ ( { next } ) => next() }
+			next="onLoaded"
+		/>
+		<Step
+			name="onLoaded"
+			wait={ () => !! query( '.plugin-item-jetpack .form-toggle:enabled' ).length }
+			target=".plugin-item-jetpack .form-toggle__switch"
```

Comment:
I added the `'plugin-item-' + plugin.slug` class to `plugin-item.jsx` (see below) so the GT could easily query the JP plugin toggle. However, there's already an `id` we might be able to use

![image](https://user-images.githubusercontent.com/96308/42696313-1ac702ac-86b8-11e8-9a7d-304fbbea612e.png)

Specifically, we might use `label[for=autoupdates-jetpack-:site] .form-toggle__switch` as selector (which means adding some more silly `:site` replacements to GT -- there's [precedent](https://github.com/Automattic/wp-calypso/blob/caff466428fb61828b4102ec9707c768ac7b2443/client/layout/guided-tours/config-elements/site-link.js#L39)).

Not sure it's worth it :man_shrugging: